### ISSUE TITLE
Suggestion for script to automatically download latest version / specific version

### ISSUE THREAD

**Issue by queglay** [ThinkboxSoftware/Deadline#48]
I had a suggestion - It would be really helpful if a script were available to download the latest version of the deadline installer / or be able to set a specific version to download for maintaining consistency in infrastructure that is provisioned in an automated fashion.

**Comment by RandomInsano** [comment]
Yes! This would be awesome. If someone would want to take this on as a project, use this as a base:
https://forums.thinkboxsoftware.com/t/deadline-docker-container/11964

You'll need to auth your user to use the AWS CLI but that would allow doing handy things. That thread has some other neat ideas in it too!
